src/encoded/static/components/regulome_search.js

```diff
@@ -478,6 +476,8 @@ export const ResultsTable = (props) => {
     let dataColumns = null;
     if (props.dataFilter === 'chromatin') {
         dataColumns = dataColumnsChromatin;
+    } else if (props.dataFilter === 'qtl' && props.shortened) {
```

Comment:
_Seems_ like you could just put the `props.shortened` test inside the test against 'qtl' to avoid the second `props.dataFilter === 'qtl'`.